recipe/meta.yaml

```diff
@@ -32,17 +32,17 @@ requirements:
         - numpy x.x
         - hdf4  # [not win]
         - hdf5 1.8.17|1.8.17.*
-        - geos
+        - geos 3.4.*
         - proj.4 4.9.3
         - xerces-c
         - libnetcdf 4.4.*
         - libdap4 3.18.*  # [not win]
         - libtiff 4.0.*
-        - libpng >=1.6.21,<1.7
+        - libpng >=1.6.23,<1.7
```

Comment:
In my test `>=1.6.23,<1.7` is OK on OS X while `>=1.6.21,<1.7` is known to fail.